File: ComplementApp.API/Dtos/DetallePlanPago.cs
using System;

namespace ComplementApp.API.Dtos
{
    public class DetallePlanPago
    {
        public int PlanPagoId { get; set; }
        public string Detalle4 { get; set; }
        public string Detalle5 { get; set; }
        public string Detalle6 { get; set; }
        public decimal ValorTotal { get; set; }
        public decimal SaldoActual { get; set; }
        public DateTime Fecha { get; set; }

    }
}

File: ComplementApp-SPA/src/app/_models/detallePlanPago.ts
export interface DetallePlanPago {
  planPagoId: number;
  terceroId: number;
  detalle4: string;
  detalle5: string;
  detalle6: string;
  detalle7: string;
  valorTotal: number;
  saldoActual: number;
  fecha: Date;
  operacion: number;
  viaticos: boolean;

  identificacionTercero: string;
  nombreTercero: string;
  viaticosDescripcion: string;
  crp: number;
  numeroPago: number;
  cantidadPago: number;
  valorFacturado: number;
  identificacionRubroPresupuestal: string;
  identificacionUsoPresupuestal: string;
  numeroRadicadoSupervisor: string;
  numeroRadicadoProveedor: string;
  fechaRadicadoSupervisor: Date;
  numeroFactura: string;
  observaciones: string;
  modalidadContrato: number;
  tipoPago: number;
  fechaRadicadoSupervisorFormato: string;
  fechaRadicadoProveedorFormato: string;
}
